### DISCUSSION TITLE
MESHCENTRAL2 DOWNLOAD

### DISCUSSION THREAD

**Discussion by mtran-88** [Ylianst/MeshCentral#6008] [General]
Hi, I am unable to download MeshCentral2 using the links in the installation and user's guide.  The link was in the MeshCentral2InstallGuide.pdf which points to [MeshCentral (meshcommander.com)] (https://www.meshcommander.com/meshcentral2) link.  Then there is the [The MeshCentral main information website is now here](https://meshcentral.com/info/) which gave an ERROR of 404 file not found.  I was not able to get the meshcentral2 which I am assuming is an upgrade to meshcentral1?  Thx in advance!

**Comment by si458** [comment]
Don't use the pdf as it's old
The website is https://meshcentral.com
Downloads are https://meshcentral.com/downloads.html
New docs are https://ylianst.github.io/MeshCentral/

P.s I'll get @Ylianst to fix the redirect of the meshcommander website

**Reply by mtran-88** [reply]
Hi, I've visited those sites, and still not able to get the meshcentral2 version.  The MeshCentral2 link still brings me to the same site, and errors out with a 404, file not found.  I've gone through some of the videos, installation, and user's guide.  I am still not sure how to get the AMT portion to show up on MeshCentral.  I'll go over the new docs, but I think those are the ones I am referencing already.  I am looking for details (step by step with pictures, :OP) on what is recommended for after you start mesh central. Details like how to get AMT to show up in the tab, what/how to config/when to use CIRA, TLS, and if those are needed.  I will reference those websites again, and see if I can get more info, but at this point, I can only get the agent portion to work properly.  Thanks for responding!  I really appreciate it!

**Reply by si458** [reply]
Meshcentral == meshcentral2

Meshcentral2 was the old name but it got renamed to meshcentral

**Reply by si458** [reply]
You can also check @Ylianst YouTube videos https://YouTube.com/MeshCentral

**Reply by mtran-88** [reply]
Hi, How would you know that it is running on 2.0 version?  I am seeing current version of 1.1.22. 
![image](https://github.com/Ylianst/MeshCentral/assets/166559124/ccf8824f-b0f2-4e43-b4fc-3c544d5c653a)

**Reply by si458** [reply]
The is no version 2.0>
The current is 1.1.22 and latest is 1.1.22

**Comment by Skyline643** [comment]
I'm having more or less the same issue. Wanted to upgrade the 1.1.48 version to version 2. Installed a clean Ubuntu machine to install version 2 and then import the database, config etc.

Now I see this topic and there's no version 2 then? It's because we want to use a DNS and a second alternateDNS name, but when canging the config file, it will not load...

**Reply by si458** [reply]
@Skyline643 the is no meshcentral2, it's just meshcentral
You can't use multiple dns names, u can only use a single dns name, the only way to use multiple dns is 1 dns for the agents and 1 dns for the web ui
